kt/godot-library/src/main/kotlin/godot/tests.kt

```diff
@@ -337,20 +335,30 @@ open class Node : Object() {
     open fun getParent(): Node? {
         TransferContext.writeArguments()
         TransferContext.callMethod(rawPtr, NODE_GET_PARENT, VariantType.OBJECT)
-        val readReturnValue = TransferContext.readReturnValue(VariantType.OBJECT)
+        val readReturnValue = TransferContext.readReturnValue(VariantType.OBJECT, true)
         return readReturnValue as Node?
     }
 
     open fun addChild(node: Node) {
         TransferContext.writeArguments(VariantType.OBJECT to node)
         TransferContext.callMethod(rawPtr, NODE_ADD_CHILD, VariantType.NIL)
-        TransferContext.readReturnValue(VariantType.NIL)
     }
 
     open fun removeChild(node: Node) {
         TransferContext.writeArguments(VariantType.OBJECT to node)
         TransferContext.callMethod(rawPtr, NODE_REMOVE_CHILD, VariantType.NIL)
-        TransferContext.readReturnValue(VariantType.NIL)
+    }
+
+    open fun getNode(nodePath: NodePath): Node? {
+        TransferContext.writeArguments(VariantType.NODE_PATH to nodePath)
+        TransferContext.callMethod(rawPtr, NODE_GET_NODE, VariantType.OBJECT)
+        return TransferContext.readReturnValue(VariantType.OBJECT, true) as Node?
```

Comment:
Why is the return type nullable here? Isn't `getNodeOrNull` for nullables and this one should crash if it's null?

Reply:
Nop, `get_node` will call `get_node_or_null` and add an addtional error print.

Reply:
I see. Thx for the clarification